nephio-ansible-install/README.md

```diff
@@ -0,0 +1,154 @@
+# ansible nephio
+
+## installation
+
+This repo provides the artifacts to install a Nephio environment using ansible to experiment with Nephio following [nephio ONE summit 2022 workshop](https://github.com/nephio-project/one-summit-22-workshop). The installation creates kind clusters, github repos and the manifests to get a base Nephio environment up an running.
+
+The installation assumes a VM is created with the following characteristics:
+
+- ubuntu 22.04LTS -> this is tested right now
+- 32G RAM, 8 vcpu -> we can change this based on the amount of kind clusters we need
+- SSH access with a SSH key is setup + username
+
+The creation of the VM is right now out of scope, but we can see what we can do going forward.
+Also we assume right now the ansible playbook is executed remote from the VM. We can see if people want to use a different approach going forward.
+
+In a local environment clone the repo in a local environment
+
+```bash
+git clone https://github.com/henderiw-nephio/nephio-ansible-install
```

Comment:
This URL needs to be replaced with `https://github.com/nephio-project/one-summit-22-workshop.git`

Reply:
fixed using: https://github.com/nephio-project/one-summit-22-workshop/pull/106/commits/a854bd5fcfaa5448a7d18b4d8742929f0fa059d6